doc/whats_new/v1.1.rst

```diff
@@ -131,6 +131,13 @@ Changelog
   Setting a transformer to "passthrough" will pass the features unchanged.
   :pr:`20860` by :user:`Shubhraneel Pal <shubhraneel>`.
 
+:mod:`sklearn.svm`
+...................
+
+- |Enhancement| :class:`svm.OneClassSVM`, :class:`svm.NuSVC`,
+  :class:`svm.NuSVR`, :class:`svm.SVC` and :class:`svm.SVR` now expose
+  `n_iter_`, the number of iterations of the libsvm optimization routine.
+  :pr:`21408` by :user:`Juan Martín Loyola <jmloyola>`.
 :mod:`sklearn.preprocessing`
 ............................
```

Comment:
This might solve your problem.
```suggestion
:mod:`sklearn.svm`
...................

- |Enhancement| :class:`svm.OneClassSVM`, :class:`svm.NuSVC`,
  :class:`svm.NuSVR`, :class:`svm.SVC` and :class:`svm.SVR` now expose
  `n_iter_`, the number of iterations of the libsvm optimization routine.
  :pr:`21408` by :user:`Juan Martín Loyola <jmloyola>`.

:mod:`sklearn.preprocessing`
............................
```